models/chainlink/polygon/chainlink_polygon_ccip_tokens_transferred_logs.sql

```diff
@@ -0,0 +1,59 @@
+{{
+  config(
+    alias='ccip_tokens_transferred_logs',
```

Comment:
You can add the schema in all config blocks now:
```suggestion
  config(
    schema='chainlink_polygon',
    alias='ccip_tokens_transferred_logs',
```